packages/ruleset/src/functions/error-response-schemas.js

```diff
@@ -84,7 +84,7 @@ function checkTraceProperty(properties, path) {
     return [
       {
         message:
-          'Error container model should contain property `trace` of type string',
+          'Error container model should contain property trace of type string',
```

Comment:
Looking at all of these is making me wonder if we should actually wrap "proper" names in some kind of quotes. I'm on the fence. Perhaps we should discuss in person